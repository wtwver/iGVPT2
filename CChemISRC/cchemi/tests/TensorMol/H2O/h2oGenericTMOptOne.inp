2
Geometry
3 0 1
 H HW HW1 HOH 1 0.417000000000 2 1 0.594398000000 -0.430112000000 -0.202093000000 1  3 1
 H HW HW2 HOH 1 0.417000000000 2 1 0.130350000000 0.936361000000 -0.671422000000 1  3 1
 O OW OW HOH 1 -0.134000000000 2 1 0.575252000000 0.493751000000 0.053280000000 2  1 1 2 1
