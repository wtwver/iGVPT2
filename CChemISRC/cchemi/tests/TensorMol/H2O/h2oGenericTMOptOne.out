-0.35218907072353
-0.28558960619470 -0.32278303439486 -0.65740655770230
Geometry
3 0 1
H     H     H     H      0  0.13942818466848  2 1   0.59418382314950 -0.42995096651999 -0.20245835456302  0 GRADIENT   -0.00000003409129 -0.00000082271958 -0.00000032683236
H     H     H     H      0  0.13942810483587  2 1   0.13032587092355  0.93594719397823 -0.67159996094205  0 GRADIENT   -0.00000099313738  0.00000088064583 -0.00000165173031
O     O     O     O      0  -0.27885628950435  2 1   0.57549030592696  0.49400377254176  0.05382331550507  0 GRADIENT    0.00000102722867 -0.00000005792625  0.00000197856268
